types/cross-spawn/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for cross-spawn 6.0
+// Project: https://github.com/moxystudio/node-cross-spawn
+// Definitions by: Alorel <https://github.com/Alorel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import {
+    ChildProcess,
+    SpawnOptions,
+    SpawnSyncOptions,
+    SpawnSyncReturns,
+    SpawnSyncOptionsWithStringEncoding,
+    SpawnSyncOptionsWithBufferEncoding
+} from 'child_process';
+
+declare function spawn(command: string, args?: string[], options?: SpawnOptions): ChildProcess;
+
+// Disable unified-signatures to have sync methods act as a 1:1 mirror to child_process.sync types
+declare namespace spawn {
+    function sync(command: string): SpawnSyncReturns<Buffer>;
+    function sync(command: string, options?: SpawnSyncOptionsWithStringEncoding): SpawnSyncReturns<string>;
+    // tslint:disable-next-line:unified-signatures
+    function sync(command: string, options?: SpawnSyncOptionsWithBufferEncoding): SpawnSyncReturns<Buffer>;
```

Comment:
This could be merged with the first signature. You don't need both `command: string` and `command: string, options?: SpawnSyncOptionsWithBufferEncoding` overloads if the same thing is returned in both situations.